#!/usr/bin/env python
PACKAGE = "impedance_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("ke1_z", double_t, 0, "Integral adaptation gain", 0.0001, 0, 10000)
gen.add("ke2_z", double_t, 0, "Proportional adaptation gain", 0.0001, 0, 10000)
gen.add("kp_z", double_t, 0, "Proportional error gain", 0.0, 0, 10000)

exit(gen.generate(PACKAGE, "impedance_control", "ImpedanceControl"))